client/pages/editntc/editntc.wxml

```diff
@@ -1,2 +1,25 @@
 <!--pages/userinfo/userinfo.wxml-->
-<text>pages/editntc/editntc.wxml</text>
+<view class='edit-notice'>
+  <view class='title'>
+    <text class='title-text'>Update training information</text>
+  </view>
+  <view class="form-container">
+    <form class="notice-form" bindsubmit="formSubmit">
+      <view class="form-element">
+        <view class="section-title">Training location</view>
+          <input name="location" placeholder="NUS" />
+      </view>
+      <view class="form-element">
+        <view class="section-title">Detailed location</view>
+        <input name="detaild" placeholder="Lower Field" />
+      </view>
+      <view class="form-element">
+        <view class="section-title">Start time</view>
+        <input name="time" placeholder="1:30pm" />
+      </view>
```

Comment:
Can you make it an date picker, and set it to next Sunday 2pm by default? Eventually we need to send the data to backend. Having a date picker makes it easier to standardize the format of data sent to the backend.